runtime/lua/vim/lsp/buf.lua

```diff
@@ -420,7 +433,7 @@ function M.signature_help(config)
     local fbuf, fwin = show_signature()
 
     if can_cycle then
-      vim.keymap.set('n', '<C-s>', function()
+      vim.keymap.set('n', '<Plug>(LspCycleSignature)', function()
```

Comment:
Let's following the same "dot" convention as we do for namespaces, autocmds, etc.
```suggestion
      vim.keymap.set('n', '<Plug>(nvim.lsp.ctrl-s)', function()
```

Reply:
`<C-s>` should still be the default. Otherwise everyone has to map it manually now?

IOW, map to both. Skip the default `<C-s>` mapping if a mapping to the `<Plug>` mapping exists.